### ISSUE TITLE
Cannot resolve directory '@angular/http' Beta 1

### ISSUE THREAD

**Issue by billfranklin** [angular/angular-cli#710]
1. OS. Windows 7 Enterprise

2. Versions
    angular-cli: 1.0.0-beta.1
    node: 5.10.1
    os: win32 x64

3. Repro steps. generated a new app and then a service, added "import { Http,Response } from '@angular/http';", WebStorm shows that it cannot resolve the directory.

4. From system-config.ts
```
const barrels: string[] = [
  // Angular specific barrels.
  '@angular/core',
  '@angular/common',
  '@angular/compiler',
  '@angular/http',
  '@angular/router',
  '@angular/platform-browser',
  '@angular/platform-browser-dynamic',


```

**Comment by billfranklin** [comment]
added   "@angular/http": "2.0.0-rc.1" to package.json and ran npm install.  Now it works.  Need to update package.json

**Comment by metric152** [comment]
I've been trying to get a PR done for just this thing.

**Comment by aciccarello** [comment]
Possibly related? #612

**Comment by aciccarello** [comment]
Also #685

**Comment by hansl** [comment]
I'm going to commit https://github.com/angular/angular-cli/pull/635 to fix this.  I kinda agree with `http` being part of the base install, but it's a slippery slope; we're going to have a lot of those packages in the angular org and we don't want to include them all in the packages (since most people wouldn't care about them).

IMO http should get an exception as it's fairly close to Angular. We might revisit later on.

**Comment by hansl** [comment]
Closed with #635 .
